vars/runBenchmarkTestScript.groovy

```diff
@@ -62,6 +63,7 @@ void call(Map args = [:]) {
             args.singleNode.toBoolean() ? "--single-node" : "",
             args.minDistribution.toBoolean() ? "--min-distribution" : "",
             args.use50PercentHeap.toBoolean() ? "--use-50-percent-heap" : "",
+            args.enableRemoteStore.toBoolean() ? "--enable-remote-store": "",
```

Comment:
Add space between `"--enable-remote-store"` and `:`.

Reply:
fixed.